src/locales/pl/PAGES/EXPLORE.json

```diff
@@ -3,7 +3,7 @@
     "INTRO": "Przejrzyj witrynę, aby zrozumieć jej cel, funkcjonalność i sposób użytkowania. Przegląd witryny pomoże Ci ustalić, które strony internetowe poddać ocenie w kolejnych krokach. Określ technologie internetowe wykorzystane do stworzenia witryny i, jeśli chcesz, zanotuj uwagi na temat innych aspektów witryny. Jeśli nie wiesz, jakie technologie zostały użyte do stworzenia witryny, zapytaj o to osoby zajmujące się witryną oraz te, które ją zaprojektowały.",
     "RESOURCE_LINK_NAME": "Krok 2: Poznaj ocenianą witrynę",
     "HD_RELIEDUP_TECH": "Zastosowane technologie internetowe",
-    "INF_RELIEDUP_TECH": "Określ, jakie technologie internetowe [obsługujące dostępność zgodnie z WCAG 2](https://www.w3.org/TR/WCAG21/#dfn-relied-upon) zastosowano na witrynie. Aby uzyskać więcej informacji, zobacz [WCAG-EM Krok 2.d: Określ technologie, na których można polegać](https://www.w3.org/TR/WCAG-EM/#step2d).",
+    "INF_RELIEDUP_TECH": "Określ, jakie technologie internetowe [obsługujące dostępność zgodnie z WCAG 2](https://www.w3.org/Translations/WCAG21-pl/#dfn-obs-ugujaca-dostepnosc) zastosowano na witrynie. Aby uzyskać więcej informacji, zobacz [WCAG-EM Krok 2.d: Określ technologie, na których można polegać](https://www.w3.org/TR/WCAG-EM/#step2d).",
```

Comment:
@zwiastunsw Problem with the link? This goes to the definition of "accessibility supported".